backend/app/tests.py

```diff
@@ -1,14 +1,49 @@
 """
-Tests for the main app.
+Tests for the gender analysis web app.
 """
 
 from django.test import TestCase
+from django.core.exceptions import ObjectDoesNotExist
+from .models import (
+    Pronoun,
+)
+
+
+class PronounTestCase(TestCase):
+    """
+    TestCase for the Pronoun model
+    """
+
+    def setUp(self):
+        Pronoun.objects.create(identifier='he', type='subj')
+        Pronoun.objects.create(identifier='him', type='obj')
+        Pronoun.objects.create(identifier='HIS', type='pos_det')
+        Pronoun.objects.create(identifier='his', type='pos_pro')
+        Pronoun.objects.create(identifier='himself', type='reflex')
+
+    def test_models_save(self):
+        he = Pronoun.objects.get(identifier='he')
+        self.assertEqual(str(he), 'Pronoun: he\nType: Subject')
+        self.assertEqual(type(he.type), str)
+        he.save()
+
+        with self.assertRaises(ObjectDoesNotExist):
+            was_converted_to_lowercase = Pronoun.objects.get(identifier='HIS')
+
+        his = Pronoun.objects.get(identifier='his', type='pos_det')
+        his_caps_until_saving = Pronoun(identifier='HIS', type='pos_pro')
+        self.assertNotEqual(his, his_caps_until_saving)
```

Comment:
in here, are you trying to test whether the program will save the case insensitive version into your database? Really thoughtful, but I think perhaps what we care more is the result after we saved them since the result is what we will be using, so perhaps the notequal is unnecessary?

Reply:
I like the impulse to shed unnecessary code. In this case, I think I'd argue that we keep the `assertNotEqual` check, as it ensures that the custom field behavior is working as intended.